### PR TITLE
Feature/days counters

### PR DESCRIPTION
**Pull request by serjKim**

### CODE REVIEW

File: mobile/src/reducers/calendar/convert-hours-credit-to-days.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+export class ConvertHoursCreditToDays {
```
Comment (santee) [inline mobile/src/reducers/calendar/convert-hours-credit-to-days.ts]: I want to see a `spec` file on this

**Comment by victor-kotov-arcadia** [comment]
re: UI lock issue environment details: iOS simulator (iPhone X, iOS 11.2). By the way: should we pay attention to specific layout issues? (UI misaligned at iPhone X)

**Comment by serjKim** [comment]
Updated. Redesign only the agenda section according to figma's sketch (var 3).
screenshot_1517845832

File: mobile/src/calendar/days-counters/days-counter-selected-day.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+import { daysCounterSelectedDayStyles } from '../styles';
+
+interface DaysCounterSelectedDayProps {
+    day: string;
+    month: string;
+}
+
+export  class DaysCounterSelectedDay extends Component<DaysCounterSelectedDayProps> {
```
Comment (santee) [inline mobile/src/calendar/days-counters/days-counter-selected-day.tsx]: why `DaysCounter`? If it's a circle in the middle, call it SelectedDay?

Comment (serjKim) [reply mobile/src/calendar/days-counters/days-counter-selected-day.tsx]: fixed

File: mobile/src/calendar/days-counters/days-counters.tsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        super(props);
+        this.state = {
+            daysCountersWidth: 0
+        };
+    }
+
+    public componentWillReceiveProps(nextProps: Readonly<DaysCountersProps>, nextContext: any) {
+        const { employee } = nextProps;
+        if (this.props.employee !== employee) {
+            this.props.calculateDaysCounters(employee.vacationDaysLeft, employee.hoursCredit);
```
Comment (santee) [inline mobile/src/calendar/days-counters/days-counters.tsx]: tell me, why is it better to call it directly here rather then react on employee change in reducer? If you forget to call this method (or you want to use somewhere else before going on this screen), your state becomes inconsistent.

File: mobile/src/calendar/days-counters/days-counters.tsx [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+            ? <DaysCounter  textValue={allVacationDays.toString()}
+                            title={allVacationDays.title} />
+            : <EmptyDaysCounter />;
+
+        const daysoffCounter = hoursCredit
+            ? <DaysCounter  textValue={hoursCredit.toString()}
+                            title={hoursCredit.title} />
+            : <EmptyDaysCounter />;
+
+        // TODO: temp
```
Comment (santee) [inline mobile/src/calendar/days-counters/days-counters.tsx]: please edit a comment to indicate what needs to be changed

File: mobile/src/calendar/days-counters/days-counters.tsx [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        const currentDate = moment();
+        const date = { day: currentDate.format('D'), month: currentDate.format('MMMM') };
+
+        return <View style={daysCountersStyles.container} onLayout={this.onDaysCountersLayout}>
```
Comment (santee) [inline mobile/src/calendar/days-counters/days-counters.tsx]: formatting...

File: mobile/src/calendar/days-counters/days-counters.tsx (hunk 4 above)
Comment (serjKim) [reply mobile/src/calendar/days-counters/days-counters.tsx]: fixed

File: mobile/src/calendar/days-counters/days-counters.tsx (hunk 5 above)
Comment (serjKim) [reply mobile/src/calendar/days-counters/days-counters.tsx]: fixed

**Pull request merged**